third_party/move/move-compiler-v2/tests/file-format-generator/bug_16543.move

```diff
@@ -0,0 +1,10 @@
+module 0xc0ffee::m {
+    struct Lazy(||) has drop;
+
+}
+
+module 0xc0ffee::n {
+    public fun test(): 0xc0ffee::m::Lazy {
+        || {}
```

Comment:
I think it would make more sense for this test to go under `tests/bytecode-generator/`? We would avoid the noise of having two exp files with the same info.

Reply:
Sure, I originally added to this folder because ICE seems to be generated from file format generator